src/Utils.ts

```diff
@@ -65,6 +65,18 @@ export default class Utils {
     return guidRegEx.test(guid);
   }
 
+  public static isPositiveInt(positiveInt: string): boolean {
+    const postitiveIntRegEx = new RegExp(/^-{0,1}\d+$/);
```

Comment:
Rather than checking if it's a valid number using a regex, we should try to parse the string as a number and if it's a number check if it's greater than zero.

Reply:
I did originally think of this, but was in a very rare RegExp kind of mood and it felt quite nice and neat for what we want it to do :)

I have now updated this to convert to a number and do the checking as suggested.  And I have used the following stackoverflow as inspiration: 
https://stackoverflow.com/questions/23437476/in-typescript-how-to-check-if-a-string-is-numeric/23440948#23440948

Reply:
FYI - I have just added in tests for the two new utils functions.  Not exhaustive, but a start.